service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfChangeController.java

```diff
@@ -52,7 +61,17 @@ public CallbackResponse handleAboutToStart(@RequestBody CallbackRequest request)
 
         caseData = getCaseData(caseDetails);
 
-        caseDetails.getData().putAll(legalCounselUpdater.updateLegalCounselFromNoC(caseData, originalCaseData));
+        ChangeOrganisationRequest nocRequest = caseData.getChangeOrganisationRequestField();
+
+        Map<String, Object> changeOrgRequestField = (Map<String, Object>) caseDetails.getData().get(
+            "changeOrganisationRequestField");
```

Comment:
think this is the same data in two different ways - probably can just use nocRequest? (and then re-use it in lines 69-74)